meson.build

```diff
@@ -1268,6 +1267,16 @@ if features['vulkan'] and features['x11']
      sources += files('video/out/vulkan/context_xlib.c')
 endif
 
+features += {'vk_khr_display': false}
+if features['vulkan']
+    features += {'vk_khr_display': cc.has_function('vkCreateDisplayPlaneSurfaceKHR', prefix: '#include <vulkan/vulkan_core.h>',
+                                                   dependencies: [vulkan])}
+endif
```

Comment:
This can just be pulled out of the conditional. Passing an empty dependencies is fine and it should just fail.